src/erc165/model/index.ts

```diff
@@ -0,0 +1,15 @@
+export class Erc165InterfaceContract {
```

Comment:
Why do we need this?

Reply:
you're right. Part of the old implementation before I have started  to use the erc165Service. Deleted.